spec/models/package_manager/go/go_mod_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe PackageManager::Go::GoMod do
+  subject(:go_mod) { described_class.new(mod_contents) }
+  let(:mod_contents) { "" }
+
+  describe "#retracted_version_ranges" do
+    context "when empty" do
+      let(:mod_contents) { "" }
```

Comment:
There's no need to set this value again as it's the same as the value on line 7.